.github/workflows/generate-token-lists.yml

```diff
@@ -149,40 +157,28 @@ jobs:
             exit 1
           fi
 
-      - name: Get new version
-        id: newVersion
-        if: ${{ matrix.commands.version == true && matrix.commands.version == true }}
-        run: |
-          version=$(cat ./src/${{ matrix.commands.paths[0] }} | jq .version | jq 'join(".")')
-          if [[ -n $version ]]; then
-            echo "newVersion=$version" >> $GITHUB_OUTPUT
-          else
-            # Make sure failure from curl or jq fails the generation
-            exit 1
-          fi
-
-      - name: Backup (Test folder)
-        if: ${{ inputs.environment == 'Test' && matrix.commands.version == true }}
+      - name: Backup
+        if: ${{ matrix.commands.version == true }}
         run: |
           paths=(${{ join(matrix.commands.paths, ' ') }})
           for path in ${paths[*]}
           do
-            aws s3 cp s3://${{ secrets.AWS_BUCKET }}/TestFolder/$path s3://${{ secrets.AWS_BUCKET }}/TestFolder/${{ steps.onlineVersion.outputs.onlineVersion }}/
+            if [[ "${{ inputs.environment }}" == "Test" ]]
+            then
+              additionalPath='TestFolder/'
+            else
+              additionalPath='/'
```

Comment:
looks like it should be an empty string instead?

Reply:
We can do both, have it as `TestFolder` or empty string. And add the slash where we use it, or have it here, right now we have it as part of the variable (see line 168)